### PR TITLE
MAUI-3165 : Add API reference links for the MAUI linear and circular progress bars UG.

### PR DESCRIPTION
**Pull request by BalachandarSelvam**
Hi @vigneshrameshsync,

I have added API reference link for linear and circular progress bar API.

Regards,
Balachandar.

### CODE REVIEW

**Comment by BalachandarSelvam** [comment]
rebuild

**Comment by BalachandarSelvam** [comment]
rebuild

File: MAUI/CircularProgressBar/Appearance.md [hunk 1]
```diff
@@ -128,13 +128,13 @@ this.Content = circularProgressBar;
...
+* [`ProgressThickness:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfCircularProgressBar.html#Syncfusion_Maui_ProgressBar_SfCircularProgressBar_ProgressThickness) Defines the thickness of the progress indicator.
+* [`TrackRadiusFactor:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfCircularProgressBar.html#Syncfusion_Maui_ProgressBar_SfCircularProgressBar_TrackRadiusFactor) Defines the outer radius of the track indicator.
+* [`TrackThickness:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfCircularProgressBar.html#Syncfusion_Maui_ProgressBar_SfCircularProgressBar_TrackThickness) Defines the thickness of the track indicator.
+* [`ThicknessUnit:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfCircularProgressBar.html#Syncfusion_Maui_ProgressBar_SfCircularProgressBar_ThicknessUnit) Specifies whether the `ProgressThickness` or `TrackThickness` are defined in pixel or factor.
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/Appearance.md:135]: API links not included for  `ProgressThickness` or `TrackThickness`

File: MAUI/CircularProgressBar/Appearance.md [hunk 2]
```diff
@@ -235,4 +235,4 @@ this.Content = circularProgressBar;
 
 ![.NET MAUI Circular ProgressBar with color customization](images/appearance/color.png)
 
-N> Refer to our `.NET MAUI Circular ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/Appearance.md:238]: [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Circular ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/CircularProgressBar/Events.md [hunk 3]
```diff
@@ -85,4 +85,4 @@ private void CircularProgressBar_ProgressCompleted(object sender, ProgressValueE
 
 {% endtabs %} 
 
-N> Refer to our `.NET MAUI Circular ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/Events.md:88]: [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Circular ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/CircularProgressBar/Range.md [hunk 4]
```diff
@@ -37,4 +37,4 @@ this.Content = circularProgressBar;
 
 ![.NET MAUI Circular ProgressBar with range customization](images/define-range/range.png)
 
-N> Refer to our `.NET MAUI Circular ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/Range.md:40]: [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Circular ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/CircularProgressBar/Segments.md [hunk 5]
```diff
@@ -60,4 +60,4 @@ this.Content = circularProgressBar;
 
 ![.NET MAUI Circular ProgressBar with segment gap width](images/segment/gap-width.png)
 
-N> Refer to our `.NET MAUI Circular ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/Segments.md:63]: [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Circular ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/CircularProgressBar/custom-content.md [hunk 6]
```diff
@@ -79,4 +79,4 @@ this.Content = circularProgressBar;
 
 ![.NET MAUI Circular ProgressBar with custom content](images/custom-content/content.png)
 
-N> Refer to our `.NET MAUI Circular ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/custom-content.md:82]: [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Circular ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/CircularProgressBar/getting-started.md [hunk 7]
```diff
@@ -185,4 +185,4 @@ SfCircularProgressBar sfCircularProgressBar = new SfCircularProgressBar
 
 N> 
 * Get the complete getting started sample from [`GitHub`](https://github.com/SyncfusionExamples/Getting-Started-with-.NET-MAUI-Circular-ProgressBar-control.-) link.
-* Refer to our `.NET MAUI Circular ProgressBar` feature tour page for its groundbreaking feature representations.
\ No newline at end of file
+* Refer to our [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations.
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/getting-started.md:188]: [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Circular ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/CircularProgressBar/getting-started.md [hunk 8]
```diff
@@ -185,4 +185,4 @@ SfCircularProgressBar sfCircularProgressBar = new SfCircularProgressBar
 
 N> 
 * Get the complete getting started sample from [`GitHub`](https://github.com/SyncfusionExamples/Getting-Started-with-.NET-MAUI-Circular-ProgressBar-control.-) link.
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/getting-started.md:187]: [`GitHub`](https://github.com/SyncfusionExamples/Getting-Started-with-.NET-MAUI-Circular-ProgressBar-control.-) -> [GitHub](https://github.com/SyncfusionExamples/Getting-Started-with-.NET-MAUI-Circular-ProgressBar-control.-)

File: MAUI/CircularProgressBar/migration.md [hunk 9]
```diff
@@ -91,112 +91,112 @@ SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
 <th>Description</th></tr>
 <tr>
 <td>{{'[Progress](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_Progress)'| markdownify }}</td>
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/migration.md:93]: .NET MAUI SfCircularProgressBar -> FT link not included in line no 12.

File: MAUI/CircularProgressBar/migration.md [hunk 10]
```diff
@@ -91,112 +91,112 @@ SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
...
 <td>{{'[ValueChanged](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_ValueChanged)'| markdownify }}</td>
-<td>{{'ProgressChanged'| markdownify }}</td>
+<td>{{'[ProgressChanged](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_ProgressChanged)'| markdownify }}</td>
 <td>The value change event occurs when the Progress is changed.</td></tr>
 <tr>
 <td>{{'[ProgressCompleted](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_ProgressCompleted)'| markdownify }}</td>
-<td>{{'ProgressCompleted'| markdownify }}</td>
+<td>{{'[ProgressCompleted](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_ProgressCompleted)'| markdownify }}</td>
 <td>The progress completed event occurs when Progress value attains Maximum value in ProgressBar.</td></tr>
 </table>
```
Comment (vigneshrameshsync) [inline MAUI/CircularProgressBar/migration.md:201]: At last line of this document, the API link not included for `Content`

File: MAUI/LinearProgressBar/Animation.md [hunk 11]
```diff
@@ -13,13 +13,13 @@ The linear progress bar provides animation support to visualize the progress val
...
+* [`SecondaryAnimationDuration:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfLinearProgressBar.html#Syncfusion_Maui_ProgressBar_SfLinearProgressBar_SecondaryAnimationDuration) Represents animation duration of the determinate state’s secondary progress indicator.
+* [`IndeterminateAnimationDuration:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_IndeterminateAnimationDuration) Represents animation duration of the indeterminate state’s indicator.
 
 ## Easing effects
 
-The `AnimationEasing` property allows you specify the transfer function that controls animation speed when they run. 
+The [`AnimationEasing`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_AnimationEasing) property allows you specify the transfer function that controls animation speed when they run. 
 
 The following code sample demonstrates the `CubicInOut` easing function of the linear progress bar.
```
Comment (vigneshrameshsync) [inline MAUI/LinearProgressBar/Animation.md:24]: API link not added.

File: MAUI/LinearProgressBar/Animation.md [hunk 12]
```diff
@@ -45,19 +45,19 @@ this.Content = linearProgressBar;
...
+N> The animation duration and easing effect parameters will not affect the configuration of the [`AnimationDuration`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_AnimationDuration) and [`AnimationEasing`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_AnimationEasing) properties.
 
 ## Indeterminate Easing Effects
 
-The `IndeterminateAnimationEasing` property allows you to specify a transfer function for indeterminate state, which controls animation speed when they run.
+The [`IndeterminateAnimationEasing`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_IndeterminateAnimationEasing) property allows you to specify a transfer function for indeterminate state, which controls animation speed when they run.
 
 The following code sample demonstrates the `BounceIn` easing function of the linear progress bar.
```
Comment (vigneshrameshsync) [inline MAUI/LinearProgressBar/Animation.md:62]: API link not included.

File: MAUI/LinearProgressBar/Animation.md [hunk 13]
```diff
@@ -83,4 +83,4 @@ this.Content = linearProgressBar;
 
 ![.NET MAUI Linear ProgressBar with indeterminate animation](images/animation/indeterminate.gif)
 
-N> Refer to our `.NET MAUI Linear ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
```
Comment (vigneshrameshsync) [inline MAUI/LinearProgressBar/Animation.md:86]: [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Linear ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/LinearProgressBar/Appearance.md [hunk 14]
```diff
@@ -218,4 +218,4 @@ this.Content = linearProgressBar;
 
 ![.NET MAUI Linear ProgressBar with seconday progress color customization](images/appearance/secondary-progress.png)
 
-N> Refer to our `.NET MAUI Linear ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
```
Comment (vigneshrameshsync) [inline MAUI/LinearProgressBar/Appearance.md:221]: [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Linear ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/LinearProgressBar/Range.md [hunk 15]
```diff
@@ -38,4 +38,4 @@ this.Content = linearProgressBar;
 
 ![.NET MAUI Linear ProgressBar with range customization](images/define-range/range.png)
 
-N> Refer to our `.NET MAUI Linear ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
```
Comment (vigneshrameshsync) [inline MAUI/LinearProgressBar/Range.md:41]: [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Linear ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/LinearProgressBar/Segments.md [hunk 16]
```diff
@@ -60,4 +60,4 @@ this.Content = linearProgressBar;
 
 ![.NET MAUI Linear ProgressBar with segment gap width](images/segment/gap-width.png)
 
-N> Refer to our `.NET MAUI Linear ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
```
Comment (vigneshrameshsync) [inline MAUI/LinearProgressBar/Segments.md:63]: [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Linear ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/LinearProgressBar/getting-started.md [hunk 17]
```diff
@@ -185,4 +185,4 @@ SfLinearProgressBar sfLinearProgressBar = new SfLinearProgressBar
 
 N> 
 * Get the complete getting started sample from [`GitHub`](https://github.com/SyncfusionExamples/Getting-started-with-.NET-MAUI-Linear-ProgressBar-control-) link.
-* Refer to our `.NET MAUI Linear ProgressBar` feature tour page for its groundbreaking feature representations.
\ No newline at end of file
+* Refer to our [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations.
```
Comment (vigneshrameshsync) [inline MAUI/LinearProgressBar/getting-started.md:188]: [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) -> [.NET MAUI Linear ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar)

File: MAUI/LinearProgressBar/getting-started.md [hunk 18]
```diff
@@ -185,4 +185,4 @@ SfLinearProgressBar sfLinearProgressBar = new SfLinearProgressBar
 
 N> 
 * Get the complete getting started sample from [`GitHub`](https://github.com/SyncfusionExamples/Getting-started-with-.NET-MAUI-Linear-ProgressBar-control-) link.
```
Comment (vigneshrameshsync) [inline MAUI/LinearProgressBar/getting-started.md:187]: [`GitHub`](https://github.com/SyncfusionExamples/Getting-started-with-.NET-MAUI-Linear-ProgressBar-control-) -> [GitHub](https://github.com/SyncfusionExamples/Getting-started-with-.NET-MAUI-Linear-ProgressBar-control-)

File: MAUI/LinearProgressBar/migration.md [hunk 19]
```diff
@@ -7,7 +7,7 @@ control: SfLinearProgressBar
 documentation: ug
 ---  
 
-# Migrate from Xamarin.Forms SfLinearProgressBar to .NET MAUI SfLinearProgressBar
+# Migrate from Xamarin.Forms to .NET MAUI SfLinearProgressBar
 
 To make the migration from the [Xamarin SfLinearProgressBar](https://www.syncfusion.com/xamarin-ui-controls/xamarin-progressbar) to `.NET MAUI SfLinearProgressBar` easier, most of the APIs from the Xamarin SfLinearProgressBar were kept in the .NET MAUI SfLinearProgressBar. However, to maintain the consistency of API naming in the .NET MAUI SfLinearProgressBar, some of the APIs have been renamed. Please find the difference in the following topics.
```
Comment (vigneshrameshsync) [inline MAUI/LinearProgressBar/migration.md:12]: FT link not included.

File: MAUI/CircularProgressBar/Appearance.md (hunk 1 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/Appearance.md]: done

File: MAUI/CircularProgressBar/Appearance.md (hunk 2 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/Appearance.md]: done

File: MAUI/CircularProgressBar/Events.md (hunk 3 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/Events.md]: done

File: MAUI/CircularProgressBar/Range.md (hunk 4 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/Range.md]: done

File: MAUI/CircularProgressBar/Segments.md (hunk 5 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/Segments.md]: done

File: MAUI/CircularProgressBar/States.md [hunk 20]
```diff
@@ -38,4 +38,4 @@ this.Content = circularProgressBar;
 
 ![.NET MAUI Circular ProgressBar in indeterminate state](images/states/circular-progressbar-indeterminate.gif)
 
-N> Refer to our `.NET MAUI Circular ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Circular ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Circular ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfCircularProgressBar in .NET MAUI.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/States.md]: done

File: MAUI/CircularProgressBar/custom-content.md (hunk 6 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/custom-content.md]: done

File: MAUI/LinearProgressBar/migration.md (hunk 19 above)
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/migration.md]: done

File: MAUI/LinearProgressBar/getting-started.md (hunk 18 above)
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/getting-started.md]: done

File: MAUI/LinearProgressBar/getting-started.md (hunk 17 above)
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/getting-started.md]: done

File: MAUI/LinearProgressBar/States.md [hunk 21]
```diff
@@ -64,4 +64,4 @@ this.Content = linearProgressBar;
 
 ![.NET MAUI Linear ProgressBar with buffer](images\states\buffer.png)
 
-N> Refer to our `.NET MAUI Linear ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/States.md]: done

File: MAUI/LinearProgressBar/Segments.md (hunk 16 above)
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/Segments.md]: done

File: MAUI/LinearProgressBar/Range.md (hunk 15 above)
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/Range.md]: done

File: MAUI/LinearProgressBar/Events.md [hunk 22]
```diff
@@ -74,4 +74,4 @@ private void LinearProgressBar_ProgressCompleted(object sender, ProgressValueEve
 
 {% endtabs %} 
 
-N> Refer to our `.NET MAUI Linear ProgressBar` feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
\ No newline at end of file
+N> Refer to our [`.NET MAUI Linear ProgressBar`](https://www.syncfusion.com/maui-controls/maui-progressbar) feature tour page for its groundbreaking feature representations. Also explore our [.NET MAUI Linear ProgressBar example](https://github.com/syncfusion/maui-demos/) that shows how to configure a SfLinearProgressBar in .NET MAUI.
```
[the remark this answers is not in the document]
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/Events.md]: done

File: MAUI/LinearProgressBar/Appearance.md (hunk 14 above)
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/Appearance.md]: done

File: MAUI/LinearProgressBar/Animation.md (hunk 13 above)
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/Animation.md]: done

File: MAUI/CircularProgressBar/migration.md (hunk 10 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/migration.md:201]: done

File: MAUI/CircularProgressBar/migration.md (hunk 9 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/migration.md:93]: done

File: MAUI/CircularProgressBar/getting-started.md (hunk 8 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/getting-started.md]: done

File: MAUI/CircularProgressBar/getting-started.md (hunk 7 above)
Comment (BalachandarSelvam) [reply MAUI/CircularProgressBar/getting-started.md]: done

**Review by vigneshrameshsync — APPROVED** [review]
The changes are fine.

File: MAUI/LinearProgressBar/Animation.md (hunk 11 above)
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/Animation.md:24]: Since its a framework API we can't add the link.

File: MAUI/LinearProgressBar/Animation.md (hunk 12 above)
Comment (BalachandarSelvam) [reply MAUI/LinearProgressBar/Animation.md:62]: Since its a framework API we cant add the link.

**Pull request merged**